src/components/tables/UsersTableColums.tsx

```diff
@@ -16,20 +18,20 @@ export function usersTableColumns(onOpenAdminModal: () => void): ColumnDef<Table
         {
             accessorKey: 'actions',
             header: 'Actions',
-            cell: () => (
+            cell: ({ row }) => (
                 <div className="flex items-center justify-center gap-x-4">
                     <button
                         type="button"
-                        onClick={onOpenAdminModal}
+                        onClick={() => onOpenAdminModal(row.original.id)}
```

Comment:
I am wondering, if the user has already been made an admin, what should be done with this button, since we don't want the user to try to make them an admin again. Should it be disabled or not visible at all?